2.segment_images/scripts/2.segment_whole_organoids.py

```diff
@@ -40,7 +40,64 @@
 print(in_notebook)
 
 
-# In[6]:
+# In[2]:
+
+
+def segment_with_diameter(
```

Comment:
Loving the definition of this function - looks like it's useful. Consider adding a software test to confirm this works with something rudimentary, like a generated egg shape, or something similar. Generally, once a capability becomes firm enough to functionalize I recommend adding at least one test. We might theorize that this could be developed at a later point, but we may forget the context of which test confirmations are most important to the work.